config/base.yaml

```diff
@@ -0,0 +1,4 @@
+application:
+  port: 3000
+  host: 0.0.0.0
+  admin_token: password
```

Comment:
Are we going to need to set a different env variable for this in fly? I think right now it's just `ADMIN_TOKEN`

Reply:
Ah yes, we would. Meant to leave that for a later PR. But the general idea would be that we could get by without specifying any ENV, as right now we need this set to run the test suite and boot the app.

Reply:
So I guess we'd need to feed the value in from the top level into something that gets passed through to the different handlers, through `Data` or something. That makes sense.

Reply:
We can tackle it later if it isn't going to impact the actual running app. What would the ENV variable need to be called?

Reply:
It would be `APP_APPLICATION__ADMIN_TOKEN` assuming we kept it as the application type token.

But that's mostly just how it was in zero2prod. We can see how we like it, might end up being more complex than needed.

Reply:
The idea being we would have things like
```
APP_DATABASE__HOST
APP_DATABASE__PORT
```